server/src/main/scala/org/http4s/server/staticcontent/FileService.scala

```diff
@@ -47,8 +47,8 @@ object FileService {
   /* Returns responses for static files.
    * Directories are forbidden.
    */
-  private def filesOnly(file: File, config: Config, req: Request): Task[Option[Response]] = Task.now {
-    if (file.isDirectory()) Some(Response(Status.Unauthorized))
+  def defaultFileCollector(file: File, config: Config, req: Request): Task[Option[Response]] = Task.now {
+    if (file.isDirectory) StaticFile.fromFile(new File(file, "index.html"), Some(req))
```

Comment:
That's an excellent question.  `None` is nice because it can chain into something else, like an automated directory listing.  Although I'd expect 403 (_not_ 401) would be the most commonly desired default here.